atest/testdata/standard_libraries/collections/list.robot

```diff
@@ -331,6 +331,11 @@ Lists Should Be Equal With Named Indices As Dictionary With Too Few Values
     ${names} =    Create Dictionary    0=a    2=c
     Lists Should Be Equal    ${L3}    ${L3B}    names=${names}
 
+Lists Should Be Equal Ignore Order    
+    ${names1} =    Create List    A    B	C	D
+    ${names2} =    Create List    D    B	C	A
```

Comment:
It looks a bit strange that the separator between arguments is different. Using four spaces consistently would be better. These values aren't names either so probably variables should be just `${list1}` and `${list2}`.

Reply:
Somehow formatting of this shown differently but I just checked checked in file here and it
shows correct.
https://user-images.githubusercontent.com/33742811/74623859-3ffbb600-516c-11ea-86ac-902a73a13862.png